tests/e2e/conftest.py

```diff
@@ -446,7 +452,7 @@ def run_cli(self, arguments: List[str], *, verbosity: int = 0) -> SysCap:
             out = proc.stdout
             err = proc.stderr
             if any(
-                " ".join(arguments).startswith(start)
+                start in " ".join(arguments)
```

Comment:
This fixes the cleanup broken because of global options passed (e.g., `neuro -q job submit`)